packages/cms/src/components/portable-text/kpi-configuration/kpi-icon-input.tsx

```diff
@@ -1,64 +1,45 @@
 import * as allIcons from '@corona-dashboard/icons';
 import { iconName2filename } from '@corona-dashboard/icons';
-import {
-  Box,
-  Button,
-  Dialog,
-  Flex,
-  Grid,
-  Radio,
-  Stack,
-  studioTheme,
-  Text,
-  TextInput,
-  ThemeProvider,
-} from '@sanity/ui';
+import { Box, Button, Dialog, Flex, Grid, Radio, Stack, studioTheme, Text, TextInput, ThemeProvider } from '@sanity/ui';
 import FormField from 'part:@sanity/components/formfields/default';
 import { PatchEvent, set, unset } from 'part:@sanity/form-builder/patch-event';
 import React, { forwardRef, useCallback, useState } from 'react';
 import { isDefined } from 'ts-is-present';
+import { colors } from '@corona-dashboard/common';
 
 const allIconsToFilename: Record<string, string> = iconName2filename;
 
-const filename2IconName = Object.keys(allIconsToFilename).reduce(
-  (aggr, key) => {
-    aggr[allIconsToFilename[key]] = key;
-    return aggr;
-  },
-  {} as Record<string, string>
-);
+const filename2IconName = Object.keys(allIconsToFilename).reduce((aggr, key) => {
+  aggr[allIconsToFilename[key]] = key;
+  return aggr;
+}, {} as Record<string, string>);
 
 export type KpiIconKey = keyof typeof allIcons;
 
 export const KpiIconInput = forwardRef((props: any, ref: any) => {
   const [isOpen, setIsOpen] = useState(false);
   const onClose = useCallback(() => setIsOpen(false), []);
   const onOpen = useCallback(() => setIsOpen(true), []);
+  const [selectedIcon, setSelectedIcon] = useState<string | undefined>(undefined);
 
   const { value, onChange, compareValue, type, markers } = props;
   const iconName = value?.length ? filename2IconName[value as any] : undefined;
 
-  const TheIcon = isDefined(iconName)
-    ? (allIcons[iconName as KpiIconKey] as allIcons.Icon)
-    : undefined;
+  const TheIcon = isDefined(iconName) ? (allIcons[iconName as KpiIconKey] as allIcons.Icon) : undefined;
```

Comment:
Might be nitpicking a bit, but `TheIcon` seems a little ambiguous to me. Something like `KpiIcon` is maybe more descriptive.
```suggestion
  const KpiIcon = isDefined(iconName) ? (allIcons[iconName as KpiIconKey] as allIcons.Icon) : undefined;
```

Reply:
I also realise that this was already the case before this branch.